src/component/teacher/Teacher.js

```diff
@@ -0,0 +1,83 @@
+import { useState, useEffect } from "react";
+import RenderLesson from "./RenderLesson";
+import Form from "./Form";
+import AddQuestion from "./AddQuestion";
+import axios from "axios";
+
+
+function Teacher() {
+  const [visible, setVisible] = useState(false);
+  const [lessons, setLessons] = useState([]);
+ 
+
+  useEffect(() => {
+    axios.get("https://ldn8-islington.herokuapp.com/lessons/")
+    .then((res) => {
+      setLessons(res.data);
+    });
+  }, []);
+
+
+  const deleteLessons = (arrLesson) => {
+ axios
+      .delete(`https://ldn8-islington.herokuapp.com/lessons/${arrLesson.id}`)
+      .then((res) => {
+        if (res.status === 200) {
+          axios
+            .get("https://ldn8-islington.herokuapp.com/lessons/")
+            .then((res) => {
+              setLessons(res.data);
+            });
+        }
+      });
+  };
+
+  return (
+    <div>
+      <h1 style={{ textAlign: "center" }}>Our Expert Teachers</h1>
+      <p className="lesson-plan">
+        Here you can find a wide range of full lesson plans to use in your
+        secondary classroom. All of our lessons are designed around themes
+        engaging and relevant to secondary learners and can be used to
+        complement your school curriculum, giving students an opportunity to
+        develop their English language and skills in motivating and enjoyable
+        ways. Written by young learner experts from around the world, our lesson
+        plans are easy to use and aim to give your students the skills and
+        confidence they need to enjoy learning English.
+      </p>
+      <div className="add-question">
+        <p onClick={() => setVisible(true)}>Add Question</p>
+
+        <p type="submit" onClick={() => setVisible(false)}>
+          Cancel
+        </p>
+      </div>
+      {visible && <AddQuestion />}
+
+      <div>
+        <div className="add-content">
+          <p onClick={() => setVisible(true)}>Add Lesson</p>
+
+          <p type="submit" onClick={() => setVisible(false)}>
+            Cancel
+          </p>
+        </div>
+        {visible && <Form setLessons ={setLessons}/>}
+
+        <div className="wrapper">
```

Comment:
Delete lines 50, 56, 60 and 66. Don't want any spaces between elements in return statement.

Reply:
Thank you @douglas86. Done!